episodes/designing-a-light-microscopy-experiment.md

```diff
@@ -0,0 +1,402 @@
+---
+title: 'Designing a light microscopy experiment'
+teaching: 30
+exercises: 10
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What are the key factors to consider when designing a light microscopy 
+experiment?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Explain the main steps of designing a light microscopy experiment
+
+- Describe a few examples of common microscopy methods e.g. widefield, confocal
+
+- Explain factors to consider when choosing a microscopy technique
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+In previous episodes we focused on the fundamentals of how images are stored and 
+displayed, with Napari as our main example. In the next few episodes, we will 
+instead focus on how to design an experiment using light microscopy and how to 
+analyse the resulting image data.
+
+## Steps to designing a light microscopy experiment
+
+As we've said throughout the course, there is no one 'right' way of handling 
+image data. It depends on the research question and project in hand, with 
+different solutions being best in different situations. The same holds true for 
+designing a light microscopy experiment - here we'll discuss some general 
+guidelines, but the specifics will depend on your particular research project.
+
+For the sake of this episode, let's imagine a scenario where we are 
+investigating the effects of a specific chemical on cells grown in culture.
+What steps would we take to design a light microscopy experiment for this?
+
+A general workflow could be:
+
+1. Define your research question
+
+2. Define what you need to observe to answer that question
+
+3. Define what you need to measure to answer that question
+
+4. Choose a light microscopy method that fits your data needs
+
+5. Choose acquisition settings that fit your data needs
+
+## Define your research question
+
+First, it's essential to clearly define your research question. What do you want 
+to find out from your microscopy experiment?
+
+For example, for our chemical example, the question could be: Does the chemical 
+affect the number, size or shape of cell nuclei over time?
```

Comment:
This is a very general research question, it's not really a hypothesis that can be tested with a microscopy experiment, I think maybe it's a research theme? For the purposes of this lesson that is probably a good thing as it allows you to be quite general in your discussion and enables the students to think of something more specific. However maybe it's worth building on it here. For me a research question would be more specific "Adding chemical X results in an increase in the number of cell nuclei over N minutes."

Reply:
I'll expand this section to give some more specific (and realistic!) examples of research questions